v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/dialectadapter/postgresql/PostgreSQLDialectAdapter.java

```diff
@@ -133,7 +133,8 @@ public ImmutableList<String> discoverTables(
       schemaDiscoveryErrors.inc();
       throw new SchemaDiscoveryException(e);
     } catch (SQLException e) {
-      logger.error("Sql exception while discovering table list for datasource={}", dataSource, e);
+      logger.error(
```

Comment:
Leave the exception out of the String, so the whole stack trace is captured instead of the exception message only, like:

```java
logger.error("Sql exception while discovering table list for datasource={}", dataSource, e);
```

Reply:
Thanks, was not aware of this. Will revert.